rosidl_generator_java/cmake/rosidl_generator_java_generate_interfaces.cmake

```diff
@@ -71,8 +71,12 @@ foreach(_abs_idl_file ${rosidl_generate_interfaces_ABS_IDL_FILES})
       "${_output_path}/${_parent_folder}/${_idl_name}_Response.java"
     )
     foreach(_typesupport_impl ${_typesupport_impls})
-      list(APPEND _generated_extension_${_typesupport_impl}_files "${_output_path}/${_parent_folder}/${_idl_name}_Request.ep.${_typesupport_impl}.cpp")
-      list(APPEND _generated_extension_${_typesupport_impl}_files "${_output_path}/${_parent_folder}/${_idl_name}_Response.ep.${_typesupport_impl}.cpp")
+      list(APPEND _generated_extension_${_typesupport_impl}_files
```

Comment:
this changes will introduce confilcts with https://github.com/osrf/ros2_java/pull/73 if not imported from that commit

Reply:
I prefer to import https://github.com/osrf/ros2_java/pull/73 if possible, which doesn't seem to be the case.

Reply:
I tried cherry-picking  osrf#73 but it doesn't apply cleanly. It seemed easier to directly address the cmake warnings here..